packages/@aws-cdk/aws-apigateway/lib/requestvalidator.ts

```diff
@@ -0,0 +1,85 @@
+import { Construct, PhysicalName, Resource } from '@aws-cdk/cdk';
+import { CfnRequestValidator, CfnRequestValidatorProps } from './apigateway.generated';
+import { IRestApi, RestApi } from './restapi';
+
+export interface IRequestValidatorRef {
```

Comment:
Rename to `IRequestValidator`. Thats our convention.

Reply:
done